src/Builder.hs

```diff
@@ -21,7 +21,7 @@ import Stage
 -- 3) Linking object files & static libraries into an executable.
 -- We have CcMode for CC and GhcMode for GHC.
 
-data CcMode = CompileC | FindCDependencies
+data CcMode = CompileC | FindCDependencies | FindMissingInclude
     deriving (Eq, Generic, Show)
```

Comment:
By the way, I think we don't need both `FindCDependencies` and `FindMissingInclude`, so at some point we'll probably drop one of them. Maybe add a TODO: `TODO: Consider merging FindCDependencies and FindMissingInclude`.